packages/dialtone-vue2/components/split_button/split_button_variants.story.vue

```diff
@@ -0,0 +1,340 @@
+<template>
+  <dt-stack
+    gap="500"
+    class="d-px8"
+  >
+    <h2>Variants</h2>
+    <table class="d-table d-bt d-bb d-bbw2 d-bc-default">
+      <thead>
+        <tr>
+          <th
+            class="d-ba d-bc-default"
+            v-text="''"
+          />
+          <th
+            v-for="importance in importances"
+            :key="importance"
+            class="d-ta-center d-br d-bc-default"
+          >
+            {{ importance }}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="kind in kinds"
+          :key="kind"
+          :class="{ 'd-bgc-contrast': kind === 'inverted' }"
+        >
+          <th
+            class="d-ta-right d-ba d-brw2 d-bc-default"
+            scope="row"
+            :class="{ 'd-bc-default-inverted d-fc-primary-inverted': kind === 'inverted' }"
+          >
+            <span
+              class="d-headline--eyebrow"
+              v-text="kind"
+            />
+          </th>
+          <td
+            v-for="importance in importances"
+            :key="`${kind}-${importance}`"
+            class="d-ta-center d-br"
+            :class="[kind === 'inverted' ? 'd-bc-default-inverted' : 'd-bc-default']"
+          >
+            <abbr
+              v-if="isInvalidCombination({ kind, importance })"
+              class="d-fc-muted d-td-none d-fs-100"
+              title="Not applicable"
+            >N/A</abbr>
+            <dt-stack
+              v-else
+              gap="500"
+              class="d-jc-center"
+            >
+              <div>
+                <dt-split-button
+                  size="xs"
+                  :kind="kind"
+                  :importance="importance"
+                  aria-label="Open dropdown"
+                >
+                  Place call
+                  <template #dropdownList>
+                    <dt-list-item
+                      v-for="item in listItems"
+                      :key="item.id"
+                      navigation-type="tab"
```

Comment:
Any particular reason you chose `tab`? I expect to be the default `arrow-keys`.

Reply:
Not really, was the first option that came to my mind, changing.